tests/test_ebuild.py

```diff
@@ -32,8 +34,13 @@ def test_simple(self):
         ebuild = self.get_ebuild()
         ebuild.add_run_depend('p2os_driver')
         got_text = ebuild.get_ebuild_text('Open Source Robotics Foundation', 'BSD')
-        with open('tests/ebuild/simple_expected.ebuild', 'r') as expect_file:
-            correct_text = expect_file.read()
+        with open('tests/ebuild/simple_expected.ebuild', 'r+') as expect_file:
+            expect_file.read(16)
+            fileUpdateStr = "# Copyright " + strftime("%Y", gmtime())
+            fileUpdateStr += expect_file.read()
+            expect_file.write(fileUpdateStr)
```

Comment:
You don't really want to write back to the file here. 

I'd suggest loading the file and then using the python regex library sub function on the read data. https://docs.python.org/2/library/re.html#re.sub

it should be something like: 

```
correct_text = re.sub('Copyright 2017', 'Copyright " + strftime("%Y", gmtime(), correct_text)
```

Reply:
Ahhh, gotcha, ty, I'll get on that